<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="setting_label">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>9</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_profile_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_user">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_income_transaction">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>189</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Income Transaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_cost_transaction">
    <property name="geometry">
     <rect>
      <x>592</x>
      <y>189</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Cost Transaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_all_transaction">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete All Transaction</string>
    </property>
   </widget>
   <widget class="QLabel" name="buttonsound_label">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>282</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Button clickSound</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_sound_label">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>282</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Message Sound</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundsound_label">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Background Sound</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_csv_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export Data To CSV File</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>282</y>
      <width>116</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="sound_btn_radio_box">
     <item row="0" column="0">
      <widget class="QRadioButton" name="btn_radio_on">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="btn_radio_off">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>718</x>
      <y>280</y>
      <width>116</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="sound_bg_radio_box">
     <item row="0" column="0">
      <widget class="QRadioButton" name="bg_on_radio">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="bg_off_radio">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>444</x>
      <y>281</y>
      <width>116</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="sound_warning_radio_box">
     <item row="0" column="0">
      <widget class="QRadioButton" name="msg_radio_on">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="msg_radio_off">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OFF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="return_setting_btn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>346</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Return To Main Page</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
